snapcraft/tests/plugins/test_kernel.py

```diff
@@ -863,15 +863,15 @@ def test_kernel_image_target_as_map(self):
         plugin = kernel.KernelPlugin('test-part', self.options,
                                      project_options)
 
-        self.assertEqual(plugin.make_targets, ['Image', 'modules'])
+        self.assertEqual(plugin.make_targets, ['Image', 'modules', 'dtbs'])
```

Comment:
Do we have any tests that cover when _not_ building on these architectures, and thus makes sure dtbs aren't built for them?